src/common/logo.c

```diff
@@ -272,6 +323,8 @@ void ffLoadLogoSet(FFconfig* config, const char* logo)
         loadGarudaLogo(&config->logo, config->colorLogo);
     else if(strcasecmp(logo, "unknown") == 0)
         loadUnknownLogo(&config->logo);
+    else if(access(logo, R_OK) == 0)
+        loadLogoFromFile(&config->logo, config->colorLogo, logo);
```

Comment:
I think we should also introduce an extra option for this. Maybe `--logo-file`? This way the user has better control about what logo is loaded. Anyway we can leave this a a fallback for `--logo`.

Reply:
Just like all flags are handled: The last one is used. So you can overwrite previous options.